src/brevitas/core/quant/float.py

```diff
@@ -23,6 +24,7 @@ def __init__(
             signed: bool,
             exponent_bit_width: int,
             mantissa_bit_width: int,
+            case_clamp_impl: Optional[nn.Module] = None,
```

Comment:
`case_clamp_impl: Optional[nn.Module] = FloatClamp` maybe?
Similar with the other None variables that are set in the init